### ISSUE TITLE
Add app KPIs to the API

### ISSUE THREAD

**Issue by onbjerg** [1Hive/apiary#91]
Each app will have a list of APIs and an overall score (the sum of each KPI).

These will be stored in the database on each app in the format:

```ts
{
  name: string,
  value: number
}
```

The GraphQL API should expose each API and the overall score. On top of this, apps should be sortable based on their overall score.

Depends on #89

**Comment by onbjerg** [comment]
No longer relevant since we're moving to The Graph (#200).
